### ISSUE TITLE
pdfbuilder.py:149 can't set attribute - using Sphinx

### ISSUE THREAD

**Issue by sgaechter** [rst2pdf/rst2pdf#844]
I always get this error if I try to generate PDF using Sphinx:

OS: Windows 10
Python 3.8.1

Used Version:
- rst2pdf==0.96
- reportlab==3.5.34
- Sphinx==2.4.1

```python
processing rst2pdf...
index
help
license
[ERROR] pdfbuilder.py:149 can't set attribute
Traceback (most recent call last):
  File "c:\python38\lib\site-packages\rst2pdf\pdfbuilder.py", line 216, in assemble_doctree
    self.env.indexentries={docname:self.env.indexentries[docname+'-gen']}
KeyError: 'index-gen'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\python38\lib\site-packages\rst2pdf\pdfbuilder.py", line 140, in write
    doctree = self.assemble_doctree(docname,title,author,
  File "c:\python38\lib\site-packages\rst2pdf\pdfbuilder.py", line 218, in assemble_doctree
    self.env.indexentries={}
AttributeError: can't set attribute
FAILED
```

Does someone else encounter the same Issues?

**Comment by vkurup** [comment]
I did have this issue, but upgrading Sphinx to 2.4.4 fixed it. (Looks like the fix might have been in 2.4.2, which was released a few days after your report)

**Comment by stephenfin** [comment]
Going to close this on the as assumption that the issue was with Sphinx and has since been resolved
